src/Orchestration/Adapter/DockerAPI.php

```diff
@@ -429,7 +473,8 @@ public function run(
 
         $vars = $parsedVariables;
 
-        // TODO: Connect to $network if not empty
+        $labels[$this->namespace.'-type'] = 'runtime';
+        $labels[$this->namespace.'-created'] = (string) time();
```

Comment:
Do we need those?

Reply:
Not sure just copied the feature of from CLI implementation

Reply:
Removed